dev-cpp/string-theory/string-theory-3.3.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} = 9999 ]]; then
```

Comment:
I don't see a 9999 ebuild here. But if you want it, SRC_URI should be here, because git version would not have it.

https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Handling_multi_live.2Frelease_ebuilds